casper/src/main/scala/coop/rchain/casper/rholang/InterpreterUtil.scala

```diff
@@ -337,52 +338,29 @@ object InterpreterUtil {
         case Seq() =>
           (RuntimeManager.emptyStateHashFixed, Seq.empty[ByteString]).pure[F]
 
-        // For single parent, get itd post state hash
+        // For single parent, get its post state hash
         case Seq(parent) =>
           BlockStore[F].getUnsafe(parent).map(_.postStateHash).map((_, Seq.empty[ByteString]))
 
-        // Multi-parent merging on finalized fringe as a base state
+        // Multi-parent merging
         case _ =>
           for {
             dag <- BlockDagStorage[F].getRepresentation
+            // TODO where to get?
+            finallyAccepted = Set.empty[DeployChainIndex]
+            finallyRejected = Set.empty[DeployChainIndex]
```

Comment:
This is something to add in upcoming PRs - finalization state has to be read from somewhere.